sky/clouds/kubernetes.py

```diff
@@ -336,6 +338,21 @@ def _make(instance_list):
 
     @classmethod
     def check_credentials(cls) -> Tuple[bool, Optional[str]]:
+        # Check dependencies for port-forward mode.
+        try:
+            networking_mode = (kubernetes_enums.KubernetesNetworkingMode.
+                               from_skypilot_config())
+        except ValueError:
+            with ux_utils.print_exception_no_traceback():
+                raise
```

Comment:
We can leave out the error handling here, as the code will not be reached?

Also, we should avoid directly raising errors in `check_credentials`. Otherwise, `sky check` may fail without showing the output of other clouds.